tests/unit/test_utils.py

```diff
@@ -99,3 +99,49 @@ def test_flatten_list(value: List, anticpated_result: List) -> None:
     """test for flatten list"""
     actual_result = utils.flatten_list(value)
     assert list(actual_result) == anticpated_result
+
+
+class HumanTimeTestData(NamedTuple):
+    """data for human time test"""
+
+    id: str
+    value: Union[int, float]
+    expected: str
+
+
+human_time_test_data = [
+    HumanTimeTestData(id="seconds", value=1, expected="1s"),
+    HumanTimeTestData(id="minutes seconds", value=60 + 1, expected="1m1s"),
+    HumanTimeTestData(id="hours minutes seconds", value=3600 + 60 + 1, expected="1h1m1s"),
+    HumanTimeTestData(
+        id="days hours minutes seconds", value=86400 + 3600 + 60 + 1, expected="1d1h1m1s"
+    ),
+]
+
+
+@pytest.mark.parametrize("data", human_time_test_data, ids=lambda data: data.id)
+def test_human_time_integer(data: List) -> None:
```

Comment:
Didn't mypy complain?
```suggestion
def test_human_time_integer(data: HumanTimeTestData) -> None:
```

Reply:
Good catch.

Because of time constraints during the initial development of ansible-navigator, mypy and type hinting wasn't covered for the tests.  No time like the present to start using it thought.